DEVELOPING.md

```diff
@@ -21,6 +21,20 @@ For developing the buildpack, `pyenv` must be set up, and the dependencies for t
 
 Beware that an older file system is also supported which includes Python 3.4, so please take that into account while introducing additional dependencies or new dependency version.
 
+### Running in `Docker`
+As an alternative to running Python on your host you can run it in Docker container.
+* set up required environment variables
+* go to `./run-locally` folder
```

Comment:
I think we should check if the `run-locally` folder is the right place / name for the Dockerfile, composefile and bash script for the development environment. I would suggest putting these in a `dev` folder instead, but that's up for discussion.

Reply:
Agree, dev is better